doc/team_practices.md

```diff
@@ -1,24 +1,72 @@
 # C2 Team Practices
 
-This document outlines best practices for the C2 team, and anyone wanting to contribute to the project.
+This document outlines best practices for the C2 team, and anyone wanting to
+contribute to the project.
 
-## Workflow
+## The Lifecycle of a Story
 
-All code changes should originate in a story on the [story tracker](https://pivotaltracker.com/n/projects/1149728).
+A story is a definable piece work to be completed, represented in the team
+tracker as a card. Bug fixes, engineering tasks, and new user features are all
+examples of stories. Stories are represented with a short, 1-2 sentence
+description. Stories that involve user-related changes are written in the
+perspective of the end user. All stories should contain details on how the
+feature can be QA’d for acceptance.
```

Comment:
how about "tested" or "evaluated" instead of "QA'd" ? Since we're defining a "story" might as well skip the jargon at first.